### ISSUE TITLE
errors when starting vim

### ISSUE THREAD

**Issue by pabra** [embear/vim-localvimrc#66]
When I run vim on my server to edit a file, I recently get this error:
```
pepp@jet:~$ LANG=C vim .ssh/authorized_keys
Error detected while processing /home/pepp/.vim/plugged/vim-localvimrc/plugin/localvimrc.vim:
line  984:
E121: Undefined variable: v:false
E15: Invalid expression: v:false
line  986:
E121: Undefined variable: s:localvimrc_python_available
E15: Invalid expression: !s:localvimrc_python_available && has("pythonx")
line  996:
E121: Undefined variable: s:localvimrc_python_available
E15: Invalid expression: !s:localvimrc_python_available && has("python")
line 1006:
E121: Undefined variable: s:localvimrc_python_available
E15: Invalid expression: !s:localvimrc_python_available && has("python3")
Press ENTER or type command to continue
```

I use this version of vim:
```
pepp@jet:~$ LANG=C vim --version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Nov 24 2016 16:43:18)
Included patches: 1-52
Extra patches: 8.0.0056
Modified by pkg-vim-maintainers@lists.alioth.debian.org
Compiled by buildd@
Huge version without GUI.  Features included (+) or not (-):
+acl             +farsi           +mouse_netterm   +syntax
+arabic          +file_in_path    +mouse_sgr       +tag_binary
+autocmd         +find_in_path    -mouse_sysmouse  +tag_old_static
-balloon_eval    +float           +mouse_urxvt     -tag_any_white
-browse          +folding         +mouse_xterm     -tcl
++builtin_terms  -footer          +multi_byte      +terminfo
+byte_offset     +fork()          +multi_lang      +termresponse
+cindent         +gettext         -mzscheme        +textobjects
-clientserver    -hangul_input    +netbeans_intg   +title
-clipboard       +iconv           +path_extra      -toolbar
+cmdline_compl   +insert_expand   -perl            +user_commands
+cmdline_hist    +jumplist        +persistent_undo +vertsplit
+cmdline_info    +keymap          +postscript      +virtualedit
+comments        +langmap         +printer         +visual
+conceal         +libcall         +profile         +visualextra
+cryptv          +linebreak       +python          +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      -lua             +rightleft       +windows
+diff            +menu            -ruby            +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
+emacs_tags      -mouseshape      -sniff           -xsmp
+eval            +mouse_dec       +startuptime     -xterm_clipboard
+ex_extra        +mouse_gpm       +statusline      -xterm_save
+extra_search    -mouse_jsbterm   -sun_workshop    -xpm
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H     -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1      
Linking: gcc   -Wl,-Bsymbolic-functions -Wl,-z,relro -Wl,--as-needed -o vim        -lm -ltinfo -lnsl  -lselinux  -lacl -lattr -lgpm -ldl    -L/usr/lib/python2.7/config-x86_64-linux-gnu -lpython2.7 -lpthread -ldl -lutil -lm -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions      
```

Please let me know if you need more information.

**Comment by embear** [comment]
It seems that v:true and v:false variables are only available for Vim version 8.0 and newer. I changed the code to use `0` and `1` instead. Can you please try latest version ce370fe23f313b62018164282279e83ea0735f81 if this fixes the problem?

**Comment by pabra** [comment]
My vim is now starting without errors again.
Thank you very much ;)
